diff --git a/node_modules/react-native-wechat-lib/android/build.gradle b/node_modules/react-native-wechat-lib/android/build.gradle
index 872fb6b..ebedc85 100644
--- a/node_modules/react-native-wechat-lib/android/build.gradle
+++ b/node_modules/react-native-wechat-lib/android/build.gradle
@@ -5,29 +5,24 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-  compileSdkVersion safeExtGet('compileSdkVersion', 23)
-  buildToolsVersion safeExtGet('buildToolsVersion', '23.0.1')
+  namespace 'com.theweflex.react'
+  compileSdkVersion safeExtGet('compileSdkVersion', 34)
+  buildToolsVersion safeExtGet('buildToolsVersion', '34.0.0')
 
   defaultConfig {
-    minSdkVersion safeExtGet('minSdkVersion', 16)
-    targetSdkVersion safeExtGet('targetSdkVersion', 22)
+    minSdkVersion safeExtGet('minSdkVersion', 24)
+    targetSdkVersion safeExtGet('targetSdkVersion', 34)
     versionCode 1
     versionName "1.0"
-    ndk {
-      abiFilters "armeabi-v7a", "x86"
-    }
   }
 }
 
-allprojects {
-  repositories {
-    jcenter()
-    maven { url "$projectDir/../../react-native/android" }
-  }
+repositories {
+  mavenCentral()
+  google()
 }
 
 dependencies {
-  api 'com.facebook.react:react-native:+'
-  api files('libs/libammsdk.jar')
-  // compile 'com.tencent.mm.opensdk:wechat-sdk-android-without-mta:+'
+  implementation 'com.facebook.react:react-native:+'
+  implementation files('libs/libammsdk.jar')
 }
diff --git a/node_modules/react-native-wechat-lib/android/src/main/AndroidManifest.xml b/node_modules/react-native-wechat-lib/android/src/main/AndroidManifest.xml
index d7e6b6e..81b4009 100644
--- a/node_modules/react-native-wechat-lib/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-wechat-lib/android/src/main/AndroidManifest.xml
@@ -1,5 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.theweflex.react">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE"/>
diff --git a/node_modules/react-native-wechat-lib/react-native.config.js b/node_modules/react-native-wechat-lib/react-native.config.js
new file mode 100644
index 0000000..4185ad9
--- /dev/null
+++ b/node_modules/react-native-wechat-lib/react-native.config.js
@@ -0,0 +1,11 @@
+module.exports = {
+  dependency: {
+    platforms: {
+      android: {
+        packageImportPath: 'import com.theweflex.react.WeChatPackage;',
+        packageInstance: 'new WeChatPackage()',
+      },
+      ios: null,
+    },
+  },
+};
